### PR TITLE
Electron/configuration

### PR DESCRIPTION
**Pull request by Mephysto666**
Hello,
I removed PORT parameter from all package.json files because the same parameter in .env files was ignored and this was confusing.

I hope it could be useful.

Very kind regards.

Meph

### CODE REVIEW

**Comment by danieltigse** [comment]
Please do a `git rebase`. We try to avoid merge commits, only the ones created on github are valids.

**Comment by Mephysto666** [comment]
Ok Daniel, next time I'll do a rebase before submit the pull request.

**Comment by Mephysto666** [comment]
I tried to rebase as you requested.
I hope I did operations in the correct way.

Bye

Meph

File: db_interface/linux_setup.sh [hunk 1]
```diff
@@ -29,6 +30,8 @@ while true; do
     esac
 done
 
+=======
+>>>>>>> df1058c7... On branch db_interface/building/arch_support
```
Comment (Hirobreak) [inline db_interface/linux_setup.sh:34]: These are unresolved merge conflicts during your rebase

File: db_interface/linux_setup.sh [hunk 2]
```diff
@@ -72,7 +75,11 @@ printf "  - Installing build dependencies... \n";
 if [ $LSB_DISTRO == $lsbDebian ]; then
   INSTALL_DEPS_ERROR=$( { sudo apt-get install gcc cmake git pkg-config sqlite3 libsqlite3-dev -y > /dev/null; } 2>&1 )
 elif [ $LSB_DISTRO == $lsbArch ]; then
+<<<<<<< HEAD
   INSTALL_DEPS_ERROR=$( { sudo pacman -S --noconfirm base-devel gcc cmake git pkg-config sqlite3 openssl tcl > /dev/null; } 2>&1 )
+=======
+  INSTALL_DEPS_ERROR=$( { sudo pacman -S --noconfirm base-devel gcc cmake git pkg-config sqlite3  > /dev/null; } 2>&1 )
+>>>>>>> df1058c7... On branch db_interface/building/arch_support
```
Comment (Hirobreak) [inline db_interface/linux_setup.sh:82]: Unresolved merge conflict as well

File: email_loading/src/components/LinkNewDeviceWrapper.js [hunk 3]
```diff
@@ -209,13 +209,15 @@ class LinkNewDeviceWrapper extends Component {
...
+      const { status, body } = await getDataReady(
+        getRecipientIdFromRemoteData()
+      );
       switch (status) {
         case DATA_READY_STATUS: {
           const { rowid } = body;
           await acknowledgeEvents({
             eventIds: [rowid],
-            recipientId: remoteData.recipientId
+            recipientId: getRecipientIdFromRemoteData()
```
Comment (Hirobreak) [inline email_loading/src/components/LinkNewDeviceWrapper.js:220]: I'm guessing this is not yours

**Pull request closed without merging**